src/benchmark_environments/classic.py

```diff
@@ -0,0 +1,32 @@
+"""Adaptation of classic Gym environments for IRL."""
+from gym.wrappers import TimeLimit
```

Comment:
Google Python style (which we claim to largely be following in the README) is to always import modules, never classes/functions: see https://google.github.io/styleguide/pyguide.html#22-imports

So:
`from gym import wrappers`
and then later:
`wrappers.TimeLimit`.

This is one of the recommendations I'm less sure about, but I think it does generally help avoid name clash.

Reply:
This seems fine to me